src/main/java/bhs/devilbotz/commands/BalancePID.java

```diff
@@ -12,13 +12,14 @@
 
 public class BalancePID extends CommandBase {
   /** Creates a new BalancePID. */
-  private final DriveTrain m_drive;
+  // TODO: Add a feedforward term to the PID controller and tune pid
+  private final DriveTrain drive;
   // private double levelAngle;
-  PIDController pid;
+  private final PIDController pid;
 
   public BalancePID(DriveTrain drive) {
     // Use addRequirements() here to declare subsystem dependencies.
-    m_drive = drive;
+    this.drive = drive;
     pid = new PIDController(0.07, 0, 0.01);
```

Comment:
The PIDController values should be defined in the Constants.java and referenced here.